### ISSUE TITLE
Feedback for SQLG2C for section 4.2 in Module 4

### ISSUE THREAD

**Issue by rgward** [microsoft/sqlworkshops#107]
Module 4 feedback for 4.2 based on this readme.md https://github.com/microsoft/sqlworkshops/blob/master/SQLGroundToCloud/sqlgroundtocloud/04-SQLServerOnTheMicrosoftAzurePlatform.md

=====

There is not enough time in this workshop to review all of the fundamentals at a deep level here. You'll receive pointers and an overview of Azure SQL platform benefits. Then, basic networking for Azure SQL will be covered. This section will go quickly in the workshop, but it's recommended to spend some time reviewing the many resources provided in this section.

I read this and wonder if someone will feel a bit off already by this. I would reword to focus on what you plan to cover but say there is reference information available if you want to go deeper

=====

Azure SQL fundamentals

After reading this section I struggled a bit to think I had just read "fundamentals". I felt I got a list and links to a series of benefits to using Azure SQL Database which is good but I personally felt like perhaps this should be called Azure SQL Benefits.

For this to truly be fundamentals for me I'm missing topics like deployment, configuration, loading data, securing, connecting, writing an application, ....

But I know this is not a deep dive. Contact me more if you want to discuss (especially becasue we then focus alot on networking)

Also Tailspin was part of 4.1 and I see it is in 4.3 but was not mentioned in 4.2?

====

without user involvement

I personally would prefer to say we provide automation for these functions without saying it doesn't involve any user interaction. For example, for performance you still need to get involved but we assist you

=====

Geo-replication by creating readable replicas of your database in the same or different data center (region).

this link only goes to auto failover groups for MI

=====

Networking fundamentals

ok, I read through this several times. I came away feeling it was deeper than just fundamentals to be honest. Anyone who is not a network expert may struggle through this section to be honest

Also, I walked away with the impression in this section that vnets are only for MI but I think we had this now for single database as well?

https://azure.microsoft.com/en-us/blog/vnet-service-endpoints-for-azure-sql-database-now-generally-available/

=====

ExpressRoute	Provides secure and private connection between Azure and on-prem

vnets already give you secure and private. Isn't ExpressRoute more about speed?

=====

**Comment by amthomas46** [comment]
Again @rgward , awesome, quality feedback that makes this content better. I really appreciate it. Look forward to learning and growing the content from other feedbacks. Thanks!

* Added more info to end of 4.1 about Tailspin's decisions, that helps lead in to 4.2
* Added new 4.2 intro: Before Tailspin Toys Gaming commits to moving to Azure SQL Managed Instance, they want to understand some of the benefits that come with running their SQL workloads in Azure.  

     In this section, we will highlight some of the key benefits of choosing Azure for your SQL workloads. If you want to go deeper, additional reference information is available. Following, some networking fundamentals will be covered, to help you understand how Azure SQL Managed Instances and Single Databases are secured.  
* Updated geo rep link
* updated without user involvement
* Updated ExpressRoute to include speed. Note FAQ what is ExpressRoute: ExpressRoute is an Azure service that lets you create private connections between Microsoft datacenters and infrastructure that’s on your premises or in a colocation facility. ExpressRoute connections do not go over the public Internet, and offer higher security, reliability, and speeds with lower latencies than typical connections over the Internet.  
* I did a lot of digging and thinking, as well as talking with others. And I think I'll remove the networking fundamentals. It's a little more deep dive and out of place with every thing else. I moved the md containing it to within my labs, just so it exists as a deep dive potential resource. I've talked with Alain more and I think I have a better understanding of it, which is always a great plus. Thanks for calling it out.  
* Changed to Azure SQL Platform Benefits and just a section calling out the highlights. If we want to change this to add more fundamentals type of stuff we can after SA, depending how it's received.  


Changes https://github.com/microsoft/sqlworkshops/compare/anna?expand=1#diff-31b39a6aaf2847c6ceb1b82aa5ada625 , will have to scroll to 4.2. Let me know if there's anything you think I haven't addressed adequately.
